search-filter/results.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * Search & Filter Pro
+ *
+ * Sample Results Template
+ *
+ * @package   Search_Filter
+ * @author    Ross Morsali
+ * @link      https://searchandfilter.com
+ * @copyright 2018 Search & Filter
+ *
+ * Note: these templates are not full page templates, rather
+ * just an encaspulation of the your results loop which should
+ * be inserted in to other pages by using a shortcode - think
+ * of it as a template part
+ *
+ * This template is an absolute base example showing you what
+ * you can do, for more customisation see the WordPress docs
+ * and using template tags -
+ *
+ * http://codex.wordpress.org/Template_Tags
+ *
+ */
+
+// If this file is called directly, abort.
+if ( ! defined('ABSPATH' ) ) {
+	exit;
+}
+
+$template_directory = get_template_directory();
+$template_url = get_template_directory_uri();
+
+global $wpdb;
+
+$query1 = <<<SQL
+select post_title from {$wpdb->prefix}posts
+where post_type='audio_post'
+and post_status='publish'
+SQL;
```

Comment:
This makes me a bit nervous 😁 Why run an SQL query to get posts?

Reply:
@comzeradd this query is the most efficient solution as it only gets the post titles needed but still makes use of wpdb functions (get_results) https://developer.wordpress.org/reference/classes/wpdb/get_results/

It's a simple but efficient query.